scripts/change-readme.js

```diff
@@ -4,16 +4,18 @@ const path = require('path')
 
 
 function changeReadme({
-  releaseChannel
+  releaseChannel,
+  npmVersion
 }) {
-  if (releaseChannel === 'dev') {
+  if (releaseChannel === 'dev' || releaseChannel === 'patch-dev') {
```

Comment:
This function should have test cases

Reply:
this function does not return anything though, what do you want to test here?

Reply:
It should test if it does what it claims it does. You can split this into two functions, one that edits a string as we want and write tests for that function and another which reads the file as a string, uses the first function (which is tested) and write back the file, this second function need not be tested.

Reply:
ah yes, will do that 👍